packages/mattermost-components/rollup.config.js

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import scss from 'rollup-plugin-scss';
+
+import ts from 'rollup-plugin-ts';
+
+import packagejson from './package.json';
+
+const externals = [
+    ...Object.keys(packagejson.dependencies || {}),
+    ...Object.keys(packagejson.peerDependencies || {}),
+    'mattermost-redux',
+    'reselect',
```

Comment:
`reselect` in particular might actually be a difficult case here since the signature of `createSelector` is different in our version because we added that extra parameter for debugging. I imagine that's going to keep causing problems as long as we keep it around. We're actually collecting (and hopefully using soon!) that data though, so it's going to be a long-running problem unless we figure out a different way to do that